### PR TITLE
Rwael/sections 183 185 188 189 191 192 193 195 202 203 205 208 209

### PR DESCRIPTION
**Pull request by Rwael**
### What?
Implemented Sections mentioned in title
### Why?
Because
### How?
Writing
### Testing?
whats' that?
### Anything Else?
closes:
#28 
#43
#62
#66
#179
#183
#186
#191
#277
#278 
#281
#285
#286

### CODE REVIEW

File: fabbwled-backend/src/main/resources/sections/181-216.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+- number: 183
+  content:
+    - text: You find a shadowy natural alcove in the rocky wall of a tunnel.
```
[the remark this answers is not in the document]
Comment (Nilstrieb) [reply fabbwled-backend/src/main/resources/sections/181-216.yaml:3]: you need to use multilines

[the remark this answers is not in the document]
Comment (Rwael) [reply fabbwled-backend/src/main/resources/sections/181-216.yaml:3]: On it

[the remark this answers is not in the document]
Comment (Rwael) [reply fabbwled-backend/src/main/resources/sections/181-216.yaml]: Done?

File: fabbwled-backend/src/main/resources/sections/181-216.yaml [hunk 2]
```diff
@@ -0,0 +1,190 @@
+- number: 183
+  content:
+    - text: |
+  You find a shadowy natural alcove in the rocky wall of a tunnel.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/181-216.yaml:4]:
```suggestion
    - text: |
        You find a shadowy natural alcove in the rocky wall of a tunnel.
```

Comment (Rwael) [reply fabbwled-backend/src/main/resources/sections/181-216.yaml]: Done

File: fabbwled-backend/src/main/resources/sections/181-216.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lion-alt) [reply fabbwled-backend/src/main/resources/sections/181-216.yaml]: 👍

**Review by Nilstrieb — APPROVED** [review]

**Review by MagischerWolf05 — APPROVED** [review]
looks good

**Review by stevan04 — APPROVED** [review]
looks good

**Review by elion004 — APPROVED** [review]
😀

**Review by adrsch04 — APPROVED** [review]
Looks good

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

**Pull request merged**